microsetta_public_api/models/_taxonomy.py

```diff
@@ -353,6 +353,51 @@ def get_group(self, ids: Iterable[str], name: str = None) -> GroupTaxonomy:
                              feature_variances=list(feature_variances),
                              )
 
+    def get_counts(self, level, samples=None) -> dict:
+        """Obtain the number of unique maximal specificity features
+
+        Parameters
+        ----------
+        level : int
+            The level to obtain feature counts for. A level of zero
+            indicates domain, one is phylum, etc.
+        samples : str or iterable of str, optional
+            The samples to collect data for. If not provided, counts are
+            derived from all samples.
+
+        Returns
+        -------
+        dict
+            The {taxon: count} observed. The count corresponds to the greatest
+            taxonomic specificity in the data. As an example, if counting at
+            the phylum level, and FOO had 2 classified genera and a classified
```

Comment:
FOO is a phylum in this example?

Reply:
Maybe we should specify that these are aggregated over all samples, and how they are aggregated. E.g., The count is the number of features corresponding to the given taxon that are present in any of the given samples.

Reply:
:+1: